tests/cephfs/snapshot_clone/snap_schedule.py

```diff
@@ -66,6 +66,19 @@ def run(ceph_cluster, **kw):
             for _ in list(range(10))
         )
         client1 = clients[0]
+        log.info("Enable Snap Schedule")
+        client1.exec_command(sudo=True, cmd="ceph mgr module enable snap_schedule")
+
+        log.info("Setting OSD config to avoid impact on snap-schedule due to Scrubbing")
```

Comment:
These params needs to be set before trying to enable snap-schedule. What the params does is to update the pg status faster than the defaults. So that if scrubbing is indeed in-progress then we are not allowed to enable snap-schedule. To check that if pg status can be updated sooner and pgs get into active +clean then it fixes the problem.